utils/build-script

```diff
@@ -700,6 +705,41 @@ the number of parallel build jobs to use""",
     a '\'"""),
         action="append", dest="extra_swift_args", default=[])
 
+    android_group = parser.add_argument_group(
+        title="Build settings for Android")
+    android_group.add_argument(
+        "--android-ndk",
+        help="An absolute path to the NDK that will be used as a libc "
+             "implementation for Android builds",
```

Comment:
This would suggest to me that I should look for a subpath of the NDK root / to manually find the libc/bionic headers within. Is that true or is this expecting the NDK root?

As an aside - ndk11b is now out (initial work was done on 10e), from what I understand the android team has now deprecated gcc in favour of clang. I've kind of lost track of what's happening with the swift android toolchain- is the current paradigm still sustainable?